pkgs/top-level/all-packages.nix

```diff
@@ -29309,6 +29309,8 @@ with pkgs;
 
   chunk = callPackage ../data/fonts/chunk { };
 
+  clarity-city = callPackage ../data/fonts/clarity-city { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file